packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -156,15 +158,6 @@ function logitBiasOfTokens(tokens: Record<string, number>) {
   );
 }
 
-/**
- * Returns true if the given model supports function calling.
- * @param model The model to check.
- * @returns True if the model supports function calling, false otherwise.
- */
-function chatModelSupportsFunctions(model: ValidChatModel) {
```

Comment:
Does it make sense for ChatModel to define whether functions are supported (defaulting to true at this point)?  I'm thinking about our llama-2 proxy use case in particular.

Reply:
I think once we point OpenAIChatModel at a non-OAI endpoint, we should take care of that on our side. But since all of OpenAI's current chat models support function calling -- IMO not at this layer.

Reply:
Does it make sense to formalize the idea of a non-OAI model that uses the same API surface then?

(I happy with whatever you decide. Just want to make sure we have a plan for this!)